logging/bin/deploy_kibana_content.sh

```diff
@@ -183,6 +183,41 @@ fi
 ./logging/bin/import_kibana_content.sh logging/kibana/tenant          cluster_admins
 
 
+
+# create the all logs RBACs
+add_notice "**Elasticsearch/Kibana Access Controls**"
+LOGGING_DRIVER=true ./logging/bin/security_create_rbac.sh _all_ _all_
+
+# Create the 'logadm' Kibana user who can access all logs
+LOG_CREATE_LOGADM_USER=${LOG_CREATE_LOGADM_USER:-false}
+if [ "$LOG_CREATE_LOGADM_USER" == "true" ]; then
+
+   if user_exists logadm; then
```

Comment:
Should we allow an override of the 'logadm' username? Could be a separate issue from this if it's too much to do/test right now.

Reply:
LOL, great minds think alike.  I started implementing that and then decided on holding off on that.  New users can be created at any time by running the user.sh script.  Since we're always creating these access controls, additional users with these rights can be created as-needed.  So, if they really hate the 'logadm' user name, they can set the flag to tell us NOT to create it and then create a comparable user with their preferred username by calling the script.  We may still decide to offer the ability to override our default name for this user at some point, but for now I don't think it's critical capability.